src/hint_cli.py

```diff
@@ -56,9 +60,51 @@ def parse(argv=None):
     return path, config, action, args
 
 
+def path_last_deploy(path):
+    return path + "/.last_deploy"
+
+
+def save_config(path, config, cfg):
+    dat = {"config": config,
+           "time": time.time(),
+           "data": cfg}
+    with open(path_last_deploy(path), "wb") as f:
+        pickle.dump(dat, f)
+
+
+def read_config(path):
+    with open(path_last_deploy(path), "rb") as f:
+        dat = pickle.load(f)
+    return dat
+
+
+def load_config(path, config=None, refresh=True):
+    if os.path.exists(path_last_deploy(path)):
+        dat = read_config(path)
+        if refresh:
```

Comment:
So this is using the config as saved to disk if refresh is false, and loading the latest configuration for that config name if refresh is true? Would it be an error if the config parameter didn't march the config value found in the file? Or is that fine because you've said you want to reload whatever's there?

Reply:
This is needlessly complicated and now we just always refresh the config, based on the `config_name` saved to disk. Further comments in the code